include/openssl/rand.h

```diff
@@ -29,6 +29,9 @@ extern "C" {
 // event that sufficient random data can not be obtained, |abort| is called.
 OPENSSL_EXPORT int RAND_bytes(uint8_t *buf, size_t len);
 
+// RAND_priv_bytes calls |RAND_bytes|.
```

Comment:
IMO: document that this is just a wrapper around `RAND_bytes` for compatibility and that consumers should be using `RAND_bytes`